### ISSUE TITLE
Null check issue in `CreateToDoList`

### ISSUE THREAD

**Issue by muddi900** [effektio/effektio#474]
I am trying to fix an issue(#473), and the linter throws this warning.

```
The property 'context' can't be unconditionally accessed because the receiver can be 'null'.
Try making the access conditional (using '?.') or adding a null check to the target ('!').
```

It occurs on line 45

**Comment by gtalha07** [comment]
> I am trying to fix an issue(#473), and the linter throws this warning.
> 
> ```
> The property 'context' can't be unconditionally accessed because the receiver can be 'null'.
> Try making the access conditional (using '?.') or adding a null check to the target ('!').
> ```
> 
> It occurs on line 45
Oh I guess we are looking at this line:
```overlay = Overlay.of(context).context.findRenderObject() as RenderBox;```
Due to new changes in Flutter 3.7.0, ```OverlayState``` doesn't require null-aware  for ```context``` so this can be fixed by upgrading your Flutter version to 3.7.0.

**Comment by gtalha07** [comment]
Closing as not bug.
